xml/System.Diagnostics/Process.xml

```diff
@@ -2150,9 +2171,33 @@ There are problems accessing the performance counter API's used to get process i
         <Parameter Name="entireProcessTree" Type="System.Boolean" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="entireProcessTree">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="entireProcessTree"><see langword="true" /> to kill the associated process and its descendants; <see langword="false" /> to kill only the associated process.</param>
+        <summary>Immediately stops the associated process, and optionally all of its child/descendent processes.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ComponentModel.Win32Exception">The associated process could not be terminated.  
+  
+ -or-  
+  
+ The process is terminating.  
+  
+ -or-  
+  
+ The associated process is a Win16 executable.</exception>
```

Comment:
Aren't this and line 2136 legacy conditions -- perhaps they should be deleted? (Are there still Win16 executables running on .NET Framework?)